### ISSUE TITLE
Feature request: Support for Linear

### ISSUE THREAD

**Issue by r0bbie** [johannesjo/super-productivity#3303]
## Problem Statement
We use Linear (https://linear.app) for issue management. It would be great if SuperProductivity supported this (similar to the Jira or GitHub integrations etc)

## :grey_question: Possible Solution
Integration with Linear supported, similarly to how the Jira, GitHub, Gitlab etc integrations already function.

**Comment by johannesjo** [comment]
I agree that this would be a nice addition. I like linear. PRs are welcome!

**Comment by cpcdoy** [comment]
Would definitely love this integration!

**Comment by cpcdoy** [comment]
Still relevant

**Comment by r0bbie** [comment]
Still relevant....

**Comment by abdelkareemkobo** [comment]
still relevant ... and really really needed

**Comment by johannesjo** [comment]
Thanks to @theBenForce this can now be closed! Thank you very much!
